CRM/Donrec/Form/Task/ContributeTask.php

```diff
@@ -30,12 +30,21 @@ function buildQuickForm() {
        'last_year'         => E::ts('last year'),
        'customized_period' => E::ts('Choose Date Range')
     );
-    $this->addElement('select', 'time_period', 'Time Period:', $options, array('class' => 'crm-select2'));
-    $this->addDateRange('donrec_contribution_horizon', '_from', '_to', E::ts('From:'), 'searchDate', TRUE, FALSE);
+    $this->addDatePickerRange(
+      'donrec_contribution_horizon',
+      E::ts('Time period'),
+      FALSE,
+      FALSE,
+      E::ts('From:'),
+      E::ts('To:'),
+      [],
+      '_to',
+      '_from',
```

Comment:
Those are `_low` and `_high` by default, thus we have to pass the traditional field names here.